modules/appeals_api/spec/requests/v2/legacy_appeals_controller_spec.rb

```diff
@@ -79,45 +90,68 @@
 
     context 'when X-VA-SSN and X-VA-File-Number are missing' do
       it 'returns a 422' do
-        get_legacy_appeals(ssn: nil, file_number: nil)
+        get_legacy_appeals
         expect(response).to have_http_status :unprocessable_entity
         json = JSON.parse(response.body)
         expect(json['errors']).to be_an Array
       end
     end
 
     context 'when receive and unusable response from Caseflow' do
+      let(:ssn) { '123445223' }
+      let(:file_number) { '222222222' }
+
       before do
         allow_any_instance_of(Caseflow::Service)
           .to(receive(:get_legacy_appeals).and_return(Struct.new(:status, :body).new(200, '<nope nope nope>')))
       end
 
       it 'returns a 502' do
-        get_legacy_appeals(ssn: '123445223', file_number: '222222222')
+        get_legacy_appeals
         expect(response).to have_http_status :bad_gateway
       end
     end
 
     context 'when receive a Caseflow 4XX response' do
       let(:status) { 400 }
       let(:body) { { this_is: 'great' }.as_json }
+      let(:ssn) { '123445223' }
+      let(:file_number) { '222222222' }
 
       before do
         allow_any_instance_of(Caseflow::Service)
           .to(receive(:get_legacy_appeals).and_return(Struct.new(:status, :body).new(status, body)))
       end
 
       it 'lets 4XX responses passthrough' do
-        get_legacy_appeals(ssn: '123445223', file_number: '222222222')
+        get_legacy_appeals
         expect(response.status).to be status
         expect(JSON.parse(response.body)).to eq body
       end
     end
+
+    context 'using the versioned namespace route' do
+      let(:ssn) { '502628285' }
+
+      it 'behaves the same as when using the original route' do
+        original_response = nil
+        VCR.use_cassette('caseflow/legacy_appeals_get_by_ssn') do
+          get_legacy_appeals
+          expect(response).to have_http_status(:ok)
+          original_response = JSON.parse(response.body)
+        end
+        VCR.use_cassette('caseflow/legacy_appeals_get_by_ssn') do
+          get_legacy_appeals('/services/appeals/legacy_appeals/v2/legacy_appeals/')
+          expect(response).to have_http_status(:ok)
+          expect(JSON.parse(response.body)).to eq(original_response)
+        end
+      end
+    end
   end
 
   private
 
-  def get_legacy_appeals(ssn: nil, file_number: nil)
```

Comment:
The keyword args here were feeling a bit unwieldy once I needed a `path` argument - I rearranged things a bit to mostly use `let` instead